src/app/scheduler/components/SchedulerCalendar.tsx

```diff
@@ -44,9 +93,21 @@ const eventStyleGetter = ({ resource }: Event) => {
 
 const CustomEvent = ({ title, event }: EventProps) => {
   return (
-    <span>
-      {title}, {event.resource && event.resource.sectionCode}, {event.resource && event.resource.location}
-    </span>
+    <Flex height="100%" direction="column">
```

Comment:
will giving this `w="100%"` fill it out all the way? I noticed in the screenshot the events have a gap on the left side

Reply:
No, but I can fix it with a bit of scss.